### DISCUSSION TITLE
Windows wsl2

### DISCUSSION THREAD

**Discussion by Gabriel-c0Nsp** [psoukie/zipchord#175] [General]
I was about to ask why I can run autohotkey scripts that works just fine on wls2, but I can't use zipchord features on it? Is there something I can do to make it work?

**Comment by psoukie** [comment]
Can you be more specific about what's not working and what error you're getting? Are you using the compiled .exe version? If you're running ZipChord as a script, make sure you have the other files the zipchord.ahk includes for its run.
